File: fueldistributionsystem/newEdition/src/main/java/com/example/demo/model/Order.java
package com.example.demo.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "Order")
public class Order {
    @Id
    private String id;
    private String type;
    private int amount;

    private String orderId;

    public Order() {
    }

    public Order(String id, String type, int amount,String orderId) {
        this.id = id;
        this.type = type;
        this.amount = amount;
        this.orderId = orderId;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }

    public String getOrderId() {
        return orderId;
    }

    public void setOrderId(String orderId) {
        this.orderId = orderId;
    }
}

File: fueldistributionsystem/dispatchlist/src/app/model/Order.ts
export class Order{
  id: string;
  type:string;
  amount:number;
  orderId: string;
  available:boolean;
  day: string;

  constructor(id: string, type: string, amount: number, orderId: string, available: boolean, day: string) {
    this.id = id;
    this.type = type;
    this.amount = amount;
    this.orderId = orderId;
    this.available = available;
    this.day = day;
  }
}
